### DISCUSSION TITLE
Error The type initializer for 'ParquetSharp.LogicalTypeFactory' threw an exception.

### DISCUSSION THREAD

**Discussion by 7Sada5916064** [G-Research/ParquetSharp#420] [Q&A]
I tried using it locally and it worked normally. However, after deploying it to the Azure function, I tried to use it and got this error: 'The type initializer for 'ParquetSharp.LogicalTypeFactory' threw an exception.'

**Comment by marcin-krystianc** [comment]
Hi, can you provide more details about the error you have seen, e.g. stack trace and detailed exception message?
